tests/jerry/regression-test-issue-1973.js

```diff
@@ -0,0 +1,96 @@
+// Copyright JS Foundation and other contributors, http://js.foundation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/* issue #1973 */
+
+assert (777E7777777777 == Infinity)
+assert (-777E7777777777 == -Infinity)
+assert (777E-7777777777 == 0)
+assert (-777E-7777777777 == -0)
+
+assert (123E1000 == Infinity)
+assert (-123E1000 == -Infinity)
+assert (123E-1000 == 0)
+assert (-123E-1000 == -0)
+assert (123E00000000000 == 123)
+assert (123E000000000001 == 1230)
+
+assert (0.00000000001E301 == 1E+290)
+assert (0.0000000001E300 == 1E+290)
+assert (0.000000001E299 == 1E+290)
+
+assert (0.1E309 == 1E308)
+assert (0.1E310 == Infinity)
+assert (-0.1E309 == -1E308)
+assert (-0.1E310 == -Infinity)
```

Comment:
this block is OK, tests the corner case of max decimal exponent, but some tests are still missing: 100e307, 10e307, 10e308, and negative counterparts.

Reply:
can we not have negative exponent tests like these also? 1e-324, 0.1e-324, 1e-325, 10e-325, 0.01e-323 + negative counterparts?

Reply:
That's true, I'll add them.